src/main/java/com/service/CreatePersonService.java

```diff
@@ -14,12 +15,18 @@
     public static final String INTEREST_INPUT = "Enter your interests *** max symbols = 250 ***";
     public static final String STATUS_INPUT = "Enter your status *** 0 for single <-> 1 for in relationship <-> 2 for married";
     public static final String INVALID_INPUT = "Invalid %s input";
-    public static final String SUCCESSFUL_INPUT = "%s successfully added";
+    public static final String SUCCESSFUL_INPUT = "%s successfully added\n";
     public static final String INTEREST_CHAR_LIMIT = "Your interests have been reduced to 250 characters";
     public static final String NAME_VALIDATION_REGEX = "[a-zA-z]{3,}";
     public static final String PATTERN_REGEXP_YEAR = "\\d{4}-[01]\\d-[0-3]\\d";
     public static final String USE_INTEGERS_ONLY = "Use integers as per description";
 
+    public static final String FIRST_QUESTION = "Select a number from 1 to 10 to assess your fairness when answering ";
+    public static final String SECOND_QUESTION = "How often do you drink? " + "\n1 - I don't drink; 2 - Not very often; 3 - Every other day; 4 - Every day";
```

Comment:
PLEASE dont use pluses

Reply:
Already know you don't prefer them. Just forgot to delete them when cleaning the code few days ago

Reply:
Double or triple check your code when submitting